lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -960,7 +961,8 @@ def set_up_course(self, **course_kwargs):
         section = ItemFactory.create(
             category='sequential',
             parent_location=chapter.location,
-            due=datetime(2013, 9, 18, 11, 30, 00)
+            due=datetime(2013, 9, 18, 11, 30, 00),
+            format='due'
```

Comment:
The (misnamed) `format` field indicates the Assignment type of the subsection - for example, "Homework", "Lab", etc.  So what is the purpose of specifying "due" here and above?